spec/api/v2/token_spec.rb

```diff
@@ -3,21 +3,55 @@
 describe '/v2/token' do
 
   describe 'get token' do
+    before :all do
```

Comment:
I would suggest we use factory girl for creation of users, as that is way more convenient rather than updating all the resource creation calls after update of a signature of a resource